core-services/src/main/java/org/zalando/nakadi/service/publishing/check/Check.java

```diff
@@ -20,6 +19,15 @@ protected List<NakadiRecordResult> processError(
             final NakadiRecord failedRecord,
```

Comment:
```suggestion
            @CheckForNull final NakadiRecord failedRecord,
```

Reply:
@tor-vs-floki could you clarify why did you suggest that ?